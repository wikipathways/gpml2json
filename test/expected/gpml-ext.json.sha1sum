3d03a7abf04f7fc442b3789817a32624229bd6b9 ?-
