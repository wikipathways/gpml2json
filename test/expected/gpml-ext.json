{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["d6efe"],"drawAs":"rect","gpmlElementName":"Pathway","height":205,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":160,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20140130","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/7cbf4605a88be2b4fa5ce9b6217e42d088060e3d8f026baa0e091cf917819a1c"},"entityMap":{"d6efe":{"id":"d6efe","textContent":"GeneProduct","wpType":"GeneProduct","x":50,"y":155,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32768,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"}}}
